magda-storage-api/src/createApiRouter.ts

```diff
@@ -29,6 +29,66 @@ export default function createApiRouter(options: ApiRouterOptions) {
         return res.status(200).send("OK");
     });
 
+    /**
+     * @apiGroup Storage
+     *
+     * @api {POST} /v0/create/bucket?bucket={bucket} Request to create a new bucket
```

Comment:
Could we do `PUT /v0/create/bucket/{bucketId}` ? Generally `PUT` is more usual if you're creating something and supplying the ID, and repeats of the same operation won't have an effect. Also while I think query params in `POST` requests is technically OK, it's a bit unusual 😬 .

Reply:
Fair enough. Consistency is key :+1: